src/exasol.c

```diff
@@ -660,8 +660,8 @@ int main(int argc, char** argv) {
 
 	wincount = get_win_count();
 
-	for (int i=0; i<40; i++) cards[i].next = C_EMPTY;
-	for (int i=0; i<15; i++) rows[i] = C_EMPTY;
+	for (int i=0; i<NUMCARDS; i++) cards[i].next = C_EMPTY;
+	for (int i=0; i<ROWCOUNT; i++) rows[i] = C_EMPTY;
```

Comment:
Huh, it seems this same error (using a magic constant rather than a `DEFINE` value) exists in the `src/main.c` file as well. Just that since numbers are the same, it does not actually manifest as a bug.

There a few other places where it sues these magic values as well.

Reply:
Sure, there are things that can be improved in the code. Like the fact that those 2 loops are present in the correct form in `init_board()` anyway. But I question the value of refactoring a project that's described as "quick, dirty". If it works properly, it's fine by me.